sass/etna.scss

```diff
@@ -517,3 +533,24 @@ Modified to match the existing site (for now)
         }
     }
 }
+
+// tna-scene-setter modifiers
+
+.tna-scene-setter {
+  &--small {
+    @include typography.relative-font-size(21);
+    font-weight: $font-weight-light;
+  }
+}
+
+// tna-useful-info component - 'Good to know' information from event page
+
+.tna-useful-info {
+  background-color: colour.brand-colour("cream");
+  color: colour.brand-colour("black");
+  padding: 1.5rem 2rem 2rem 2rem;
+
+  .tna-template--pink-accent & {
+    background-color: colour.brand-colour("pastel-pink");
```

Comment:
This also might be able to use the `@include colour.light-accent-background;` and then have no need for the override?

Reply:
Excellent thanks, glad I was able to get rid of this modifier, didn't love it